src/main/java/edu/harvard/iq/dataverse/api/Users.java

```diff
@@ -260,5 +261,26 @@ public Response getTracesElement(@Context ContainerRequestContext crc, @Context
             return ex.getResponse();
         }
     }
-
+    @GET
+    @AuthRequired
+    @Path("{identifier}/allowedcollections/{permission}")
+    @Produces("text/csv, application/json")
```

Comment:
Are we purposefully returning CSV as an option? Do we have a use case? All the tests seem to be for JSON.

Reply:
just json. I'll remove it